feat: add client-side Logger (#42)
guardian/libs: src/logger.README.md

# `logger`

> Team-specific production logging API. Messages will be logged to the console

Reviewer: ```suggestion
Selectively log team-specific messages to the console.
```

only if the `gu.logger` local storage item contains the team name.

> A developer can register to multiple teams by comma-separating them.

Reviewer: ```suggestion
You can subscribe to multiple teams by comma-separating them.
```
